airflow/jobs/scheduler_job.py

```diff
@@ -970,48 +971,61 @@ def _start_queued_dagruns(
         session: Session,
     ) -> int:
         """Find DagRuns in queued state and decide moving them to running state"""
-        dag_runs = self._get_next_dagruns_to_examine(State.QUEUED, session)
-
-        active_runs_of_dags = defaultdict(
-            lambda: 0,
-            session.query(DagRun.dag_id, func.count('*'))
-            .filter(  # We use `list` here because SQLA doesn't accept a set
-                # We use set to avoid duplicate dag_ids
-                DagRun.dag_id.in_(list({dr.dag_id for dr in dag_runs})),
-                DagRun.state == State.RUNNING,
-            )
-            .group_by(DagRun.dag_id)
-            .all(),
+        active_dags = (
+            session.query(DM)
+            .filter(DM.is_active == expression.true(), DM.is_paused == expression.false())
```

Comment:
I think this can just be `DM.is_active == True, DM.is_paused == False`, but no hard preferences on this (the `==` boolean call is very ugly anyway).

Reply:
Yeah, I did that but unfortunately, it's not passing the CI

Reply:
Yeah that is mainly for MSSQL

Reply:
Do you just need count of those DAGs @ephraimbuddy not actual Dag Models? If so use func.count:

```
session.query(func.count(DM.dag_id).filter(DM.is_active == expression.true(), DM.is_paused == expression.false())).scalar()
```